libs/BLEKit/include/BLEServiceMonitoring.h

```diff
@@ -29,12 +29,21 @@ class BLEServiceMonitoring : public interface::BLEService
 		sendData(data);
 	}
 
-	auto isScreensaverEnable() const -> bool { return screensaver_enable; }
+	void resetGoToSleep()
```

Comment:
I'm not sure I understand what "reset go to sleep" actually mean in terms of what the method is doing.